File: 02_ProgrammingFundamentals/_03_Arrays/_01_Lecture/_02_PrintNumbersInReverseOrder.java
package _02_ProgrammingFundamentals._03_Arrays._01_Lecture;

import java.util.Scanner;

public class _02_PrintNumbersInReverseOrder {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Add variable
        int[] numbers = new int[n];

        // Reverse the array
        for (int i = n - 1; i >= 0; i--) {
            int number = Integer.parseInt(scan.nextLine());
            numbers[i] = number;
        }
        
        // Print result
        for (int num: numbers) {
            System.out.print(num + " ");
        }
        
        scan.close();
    }
}

File: 24_C#Fundamentals/_03_Arrays/_01_Lecture/_02_PrintNumbersInReverseOrder/_02_PrintNumbersInReverseOrder.cs
using System;

public class _01_HelloSoftUni
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Add variable
        int[] numbers = new int[n];

        // Reverse the array
        for (int i = n - 1; i >= 0; i--)
        {
            int number = int.Parse(Console.ReadLine());
            numbers[i] = number;
        }

        // Print result
        foreach (int num in numbers)
        {
            Console.Write(num + " ");
        }
    }
}
